### ISSUE TITLE
Missing uses cases for scanning codes from the display

### ISSUE THREAD

**Issue by leolux** [w3c-webmob/wake-lock-use-cases#28]
I think there are some use cases which would require the screen lock API from turning off the display while facing the display to the physical bottom. Many devices implement the additional feature to turn of the display as soon as the phone gets turned over. But this "standard" behaviour of the operation system is preventing the following use case to succeed without the sreen lock:

![scanning](https://cloud.githubusercontent.com/assets/5385814/17049724/4c7d1d2e-4fef-11e6-8d04-5b1adcb6fbd8.jpg)

Such scanning devices could be used for identification, authorization, as well as payments.

**Comment by marcoscaceres** [comment]
Agree, this is a good use case (and we should add it to the use cases doc) - but it's independent of physical orientation, in that one could simply request a screen lock and all this would work. 

@leolux, where did that photo come from?

**Comment by leolux** [comment]
Thanks!
The photo is just a random image I found while searching for such use cases. The photo actually comes from the site https://www.dm.de/alverde-magazin/dm-erleben/payback-app-bargeldlos-bezahlen-c826460.html However, I am an independent developer and I have no interests with any company.

You said that the screen lock works independently of the physical orientation. Does it mean that the screen lock always has a higher priority than the manufacturer dependent "turn over rule"?

**Comment by marcoscaceres** [comment]
On August 4, 2016 at 12:43:15 AM, leolux (notifications@github.com) wrote:

> You said that the screen lock works independently of the physical orientation. Does  
> it mean that the screen lock always has a higher priority than the manufacturer dependent  
> "turn over rule"?

If the lock is enabled and the OS permits overriding the lock, then, in theory, yes.
